build.zig

```diff
@@ -56,7 +56,7 @@ pub fn build(b: *std.build.Builder) !void {
             );
 
             if (filter == null or exe.inner.target.cpu_arch.? == filter.?) {
-                exe.inner.setBuildMode(mode);
+                exe.inner.optimize = optimize;
```

Comment:
Could you instead add `optimize` to the build options for `addEmbeddedExecutable()`? And default it to debug.

Reply:
Oh you can nuke it np